history/taskana-simplehistory-rest-spring/src/main/java/pro/taskana/simplehistory/rest/assembler/TaskHistoryEventListResourceAssembler.java

```diff
@@ -4,53 +4,57 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
 import org.springframework.hateoas.IanaLinkRelations;
 import org.springframework.hateoas.Link;
 import org.springframework.hateoas.PagedModel.PageMetadata;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+import org.springframework.web.util.UriComponentsBuilder;
 
-import pro.taskana.resource.rest.AbstractRessourcesAssembler;
 import pro.taskana.simplehistory.impl.HistoryEventImpl;
 import pro.taskana.simplehistory.rest.TaskHistoryEventController;
 import pro.taskana.simplehistory.rest.models.TaskHistoryEventListResource;
 import pro.taskana.simplehistory.rest.models.TaskHistoryEventRepresentationModel;
 
 /** Mapper to convert from a list of HistoryEventImpl to a TaskHistoryEventResource. */
-public class TaskHistoryEventListResourceAssembler extends AbstractRessourcesAssembler {
+public class TaskHistoryEventListResourceAssembler {
```

Comment:
I understand that we do not change the history events to the representation model structure in this PR? Is there a ticket for it to do it later on?